components/TableColumns/TableColumns.tsx

```diff
@@ -90,6 +105,287 @@ export const columns: ColumnDef<Header>[] = [
      */
     cell: ({ row }) => <div>{row.getValue('text3')}</div>,
   },
+  {
+    accessorKey: 'text4',
+
+    /**
+     * Value of row.
+     * @param {object} column - The column data.
+     * @param {object} column.column - The column definition
+     * @returns {JSX.Element} - The cell component.
+     */
+    header: ({ column }): JSX.Element => {
+      return (
+        <Button
+          variant="ghost"
+          onClick={() => column.toggleSorting(column.getIsSorted() === 'asc')}
+        >
+          Surviving?
+          <ChevronsUpDown className="ml-2 h-4 w-4" />
+        </Button>
+      );
+    },
+
+    /**
+     * Value of row.
+     * @param {object} row - The row data.
+     * @param {object} row.row - The row definition
+     * @returns {JSX.Element} - The cell component.
+     */
+    cell: ({ row }) => <div>{row.getValue('text2')}</div>,
+  },
+  {
+    id: 'actions',
+
+    /**
+     * Admin action dropdown.
+     * @returns {JSX.Element} - The cell component.
+     */
+    cell: (): JSX.Element => {
+      return (
+        <DropdownMenu>
+          <DropdownMenuTrigger asChild>
+            <Button variant="ghost" className="h-8 w-8 p-0">
+              <span className="sr-only">Open menu</span>
```

Comment:
Q: sr-only is meant for screen reader, correct? But based on what your text says here, I'm not sure I understand why we need to nest a span tag inside this Button

Reply:
this was also specifically created by shadcn to be used with the table.

Reply:
@choir27 To answer your question, the `<MoreHorizontal />` icon being used in the button is not descriptive enough, so shadcn/ui is using a `span` with the `sr-only` class to make it more accessible.